### DISCUSSION TITLE
University domain not recognised on Azure Student Portal

### DISCUSSION THREAD

**Discussion by fahim-ux** [community/community#143421] [GitHub Education]
### Hi everyone,

Hello Everyone
I m trying to claim my azure student benefits but my university domain is not being recognised but i have github student developer pack

**Comment by davevad93** [comment]
Hi @fahim-ux ,

This is a known problem #131336, #135739, #140395, #140678, to solve it do the following:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I personally recommend the first option.
